js/security-headers.js

```diff
@@ -0,0 +1,33 @@
+import env from "../config/env.generated.json";
+
+export default {
+  directives: {
+    defaultSrc: [
+      `'none'`
+    ],
+    scriptSrc: [
+      `'self'`,
+      `'unsafe-inline'`
+    ],
+    fontSrc: [
+      `'self'`,
+      `https://code.cdn.mozilla.net`
+    ],
+    styleSrc: [
+      `'self'`,
+      `'unsafe-inline'`,
+      `https://code.cdn.mozilla.net`
+    ],
+    imgSrc: [
+      `'self'`,
+      `data:`,
+      `*`
```

Comment:
@Pomax we need to have `*` there (at least for now) so we can have entry thumbnail show. I can remove `'self'` and `data:` from the list.